modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -1882,11 +1882,34 @@ <h4><a id="imageview">ImageView</a></h4>
       </thead>
       <tbody>
         <tr>
-        <th class="propertyname" scope="row">-fx-image</th>
+          <th class="propertyname" scope="row">-fx-fit-height</th>
+          <td class="value"><a href="#typenumber" class="typelink">&lt;number&gt;</a></td>
+          <td>0</td>
```

Comment:
Since you don't specify a style class for the newly added properties, they will all be left justified. The existing `null` value for `-fx-image` specifies `class="default"` which centers it. I recommend either adding the `class="default"` to all of the newly added properties, or removing it from the default value of the existing `fx-image` property.

I'll reapprove if you want to make the change.